src/plugins/exitscript/exitscript.js

```diff
@@ -0,0 +1,166 @@
+/**
+* @title WET-BOEW Exit script plugin
+* @overview Plugin redirects users to non secure site
+* @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+* @author @ipaksc
+*/
+( function( $, window, wb ) {
+"use strict";
+var componentName = "wb-exitscript",
+	selector = "." + componentName,
+	initEvent = "wb-init" + selector,
+	$document = wb.doc,
+	defaults = {},
+	i18nDict = {
+		en: {
+			"exitscript-warning": "Warning",
+			"exitscript-exitMsg": "You are about to leave a secure site, do you wish to continue?",
+			"exitscript-yesMsg": "Yes",
+			"exitscript-cancelMsg": "Cancel"
+		},
+		fr: {
+			"exitscript-warning": "Avertissement",
+			"exitscript-exitMsg": "Vous êtes sur le point de quitter un site sécurisé. Voulez-vous continuer?",
+			"exitscript-yesMsg": "Oui",
+			"exitscript-cancelMsg": "Annuler"
+		}
+	},
+
+/**
+* @method init
+* @param {jQuery Event} event Event that triggered the function call
+*/
+	init = function( event ) {
+		var elm = wb.init( event, componentName, selector ),
+			$elm,
+			settings;
+		if ( elm ) {
+			$elm = $( elm );
+			settings = $.extend(
+		true,
+		defaults, {},
+		window[ componentName ],
+		wb.getData( $elm, componentName )
+	);
+			i18nDict = i18nDict[ wb.lang || "en" ];
+			i18nDict = {
+				warning: i18nDict[ "exitscript-warning" ],
+				exitMsg: i18nDict[ "exitscript-exitMsg" ],
+				yesMsg: i18nDict[ "exitscript-yesMsg" ],
+				cancelMsg: i18nDict[ "exitscript-cancelMsg" ]
+			};
+			
+			wb.ready( $elm, componentName );
+		}
+
+	};
+
+	// Add your plugin event handler
+	$document.on( "click", selector, function( ev ) {
+		ev.preventDefault();
+		var elm = event.currentTarget,
+				$elm = $( elm ),
+				moDalId,
+				wrapper,
+				exitURLsearchParam,
+				exitURL,
+				exitParam = encodeURIComponent( "targetUrl=" + String( this.href ) );
+		var moDal = document.createDocumentFragment();
+		var tpl = document.createElement( "div" );
+		var settings = $.extend(
+		defaults, {}
+		);
+
+		if(this.hasAttribute("target")){ 
+		  exitParam = exitParam + encodeURIComponent( "&targetBlank=true" );
+		}
+			 
+		exitURL = settings.exitURL;
+			
+		if( !exitURL.includes( "targetUrl" ) ){ 
```

Comment:
What is the purpose of this conditional? What is "targetUrl" means, is it another inner setting?

If this condition means no code processing should continue to be executed, simply return "true" instead of closing the conditional at the end of this JS function. ex:
```
if (condition) {
    return true;
}
```